deployment/ansible/roles/model-my-watershed.rwd/tasks/main.yml

```diff
@@ -0,0 +1,17 @@
+---
+- stat: path={{ rwd_data_path }}
+  register: rwd_data
+- fail: msg="Data folder does not exist. Did you forget to mount /opt/rwd-data?"
```

Comment:
I thought we could use a synced folder for local development, and the block device should be mounted here during the packer job (see https://github.com/WikiWatershed/model-my-watershed/pull/1073/files#diff-deda55c5935c68f390c8525df625393dR343). Right?